src/main/java/org/gridsuite/cases/importer/job/CaseImportLogger.java

```diff
@@ -6,64 +6,112 @@
+    private Connection conn;
 
-    private static final String FILENAME_COLUMN = "filename";
-    private static final String ORIGIN_COLUMN = "origin";
-    private static final String IMPORT_DATE_COLUMN = "import_date";
+    private PostgresConnector postgresConnector = new PostgresConnector();
+    private String url;
+    private String username;
+    private String password;
 
-    private PreparedStatement psInsertImportedFile;
-    private String keyspaceName;
+    public void connectDBLiquibase(String url, String username, String password) {
+        this.url = url;
+        this.username = username;
+        this.password = password;
 
-    private void setKeyspaceName(String keyspaceName) {
-        this.keyspaceName = keyspaceName;
+        try {
+            conn = postgresConnector.connect(url, username, password);
+            Database database = DatabaseFactory.getInstance().findCorrectDatabaseImplementation(new JdbcConnection(conn));
+            updateLiquibase(database);
+        } catch (DatabaseException e) {
+            throw new RuntimeException(e);
+        }
     }
 
-    public void connectDb(String hostname, int port, String datacenter, String keyspaceName) {
-        connector.connect(hostname, port, datacenter);
-
-        setKeyspaceName(keyspaceName);
-
-        psInsertImportedFile = connector.getSession().prepare(insertInto(this.keyspaceName, FILES_TABLE)
-                .value(FILENAME_COLUMN, bindMarker())
-                .value(ORIGIN_COLUMN, bindMarker())
-                .value(IMPORT_DATE_COLUMN, bindMarker())
-                .build());
-
+    private void updateLiquibase(Database database) {
+        Properties properties = new Properties();
+        try (Liquibase liquibase = new Liquibase("db/changelog/db.changelog-master.yaml",
+                                                 new ClassLoaderResourceAccessor(),
+                                                 database);) {
+            properties.forEach((key, value) -> liquibase.setChangeLogParameter(Objects.toString(key), value));
+            liquibase.update(new Contexts(), new LabelExpression());
+        } catch (LiquibaseException e) {
+            throw new RuntimeException(e);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
     }
 
     public boolean isImportedFile(String filename, String origin) {
-        ResultSet resultSet = connector.getSession().execute(selectFrom(this.keyspaceName, FILES_TABLE)
-                .columns(
-                        FILENAME_COLUMN,
-                        ORIGIN_COLUMN,
-                        IMPORT_DATE_COLUMN)
-                .whereColumn(FILENAME_COLUMN).isEqualTo(literal(filename))
-                .whereColumn(ORIGIN_COLUMN).isEqualTo(literal(origin))
-                .build());
-        Row one = resultSet.one();
-        return one != null;
+        PreparedStatement preparedStatement = null;
+        try {
```

Comment:
maybe try with ressource would be better
try (PreparedStatement preparedStatement = null;
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html